stackslib/src/burnchains/burnchain.rs

```diff
@@ -211,10 +211,26 @@ impl BurnchainStateTransition {
             })
             .epoch_id;
 
+        // what was the epoch at the start of this window?
+        let window_start_epoch_id = SortitionDB::get_stacks_epoch(
+            sort_tx,
+            parent_snapshot
+                .block_height
+                .saturating_sub(epoch_id.mining_commitment_window().into()),
+        )?
+        .unwrap_or_else(|| {
+            panic!(
+                "FATAL: no epoch defined at burn height {}",
+                parent_snapshot.block_height + 1
```

Comment:
Shouldn't this be `parent_snapshot.block_height - epoch_id.mining_commitment_window().into()`?

Reply:
Yes, thanks